packages/infra/lib/api-stack.ts

```diff
@@ -428,6 +429,7 @@ export class APIStack extends Stack {
       },
       cookieStore,
     });
+    const apiLoadBalancerAddress = apiLoadBalancer.loadBalancerDnsName;
```

Comment:
Differentiate between LB address (needed for API GW integration) and "direct" access address that points to the ALB (so lambdas don't need the additional network hop of going through the NLB to get to the API).